$NetBSD: patch-ak,v 1.3 2001/08/09 13:49:00 wennmach Exp $

Do not print a cryptic message after xfs_mod has been modloaded, say it
in clear text.

--- xfs/bsd/xfs_wrap-bsd.c.orig	Tue May  9 05:17:00 2000
+++ xfs/bsd/xfs_wrap-bsd.c	Mon Nov  6 16:08:55 2000
@@ -36,6 +36,8 @@
  * SUCH DAMAGE.
  */
 
+#include <arla-version.h>
+
 #include <xfs/xfs_locl.h>
 #include <xfs/xfs_message.h>
 #include <xfs/xfs_fs.h>
@@ -308,8 +310,7 @@
     ret = lkmdispatch(lkmtp, cmd);
     if(cmd == LKM_E_LOAD) {
 	xfs_dev_major = _module.lkm_offset;
-	printf ("xfs: cdev: %d, syscall: %d\n",
-		xfs_dev_major, xfs_syscall_num);
+	printf("xfs (%s): using cdev %d, syscall %d\n", arla_version, xfs_dev_major, xfs_syscall_num);
     }
     return ret;
 }
